v2/svid/jwtsvid/mint.go

```diff
@@ -0,0 +1,14 @@
+package jwtsvid
+
+import (
+	"crypto"
+	"time"
+
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+)
+
+// Mint signs a new JWT-SVID. In accordance with the JWT-SVID specification,
```

Comment:
Maybe link to https://github.com/spiffe/spiffe/blob/master/standards/JWT-SVID.md?

Reply:
@evan2645 and I talked IRL and decided to drop `Mint` for the time being